src/main/frontend/db/react.cljs

```diff
@@ -48,8 +49,9 @@
 
 ;; Query atom of map of Key ([repo q inputs]) -> atom
 ;; TODO: replace with LRUCache, only keep the latest 20 or 50 items?
+;; WIP: use a lru cache with very low capacity
 
-(defonce query-state (atom {}))
+(defonce query-state (atom (cache/lru-cache-factory {} :threshold 1)))
```

Comment:
We can remove this since it's unrelated to the fix and isn't fixing the out of memory error